### ISSUE TITLE
tf.losses.cosineDistance treats axis as optional but not documented as such

### ISSUE THREAD

**Issue by ToonTalk** [tensorflow/tfjs#852]
https://js.tensorflow.org/api/latest/#losses.cosineDistance

Also it would be great if the description of 'axis' was more informative.

**Comment by rthadur** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
